### PR TITLE
Project 1 : E-Commerce - Zapatoos

### PR DESCRIPTION
**Pull request by FredoSalat**

### CODE REVIEW

**Comment by FredoSalat** [comment]
Netlify link: https://incandescent-choux-89829a.netlify.app/

File: style.css [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+        position: absolute;
+        cursor: pointer;
+        right: 5%;
+        top: 50%;
+        transform: translate(-5%, -50%);
+        z-index: 2;
+        transition: all 0.7s ease;
+    }
+
+    .nav-links {
```
Comment (slmgnm) [inline style.css:168]: check the nav links, they don't show on burger menu.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    <!-- navbar/ header begins  -->
+    <header>
+        <div class="hamburger">
+            <div class="line1"></div>
+            <div class="line2"></div>
+            <div class="line3"></div>
+        </div>
+
+        <h1 class="logo">Zapatoos</h1>
```
Comment (slmgnm) [inline index.html:33]: You can add a link to homepage on your logo

**Review by slmgnm — COMMENTED** [review]
Good work, responsive products section and header/footer.
still missing cart page with detailed item, login and sign up forms and detailed product's page.

**Pull request closed without merging**